customModules/config.ts

```diff
@@ -293,8 +301,7 @@ export const CustomModuleSettings = (): Scrollable<GtkWidget, Attribute> =>
                 Option({
                     opt: options.bar.customModules.kbLayout.icon,
                     title: 'kbLayout Icon',
```

Comment:
```suggestion
                    title: 'Keyboard Layout Icon',
```
Hijacking your PR to improve this name :)